driver-core/src/test/java/com/datastax/driver/core/ControlConnectionTest.java

```diff
@@ -69,7 +69,7 @@ public long nextDelayMs() {
         };
 
         try {
-            ccm = CCMBridge.builder("test").withNodes(2).build();
```

Comment:
This test could be expressed using CCMTestsSupport couldn't it?  Doesn't need to be done here, but just thinking if it is possible, I think it should be able to with

 ```java
@CCMTestMode(PER_METHOD)
....
// For each method
@CCMConfig(numberOfNodes=X, createCluster=false)
```